File: testresources/java/accuracy/src/domain/direct/violating/CallInstanceInnerInterface.java
package domain.direct.violating;

import technology.direct.dao.CallInstanceOuterClassDAO;

public class CallInstanceInnerInterface {

	protected CallInstanceOuterClassDAO.CallInstanceInnerInterfaceDAO innerInterfaceDao;

	public CallInstanceInnerInterface(){};
	
	public void test() {
		innerInterfaceDao.InterfaceMethod();
	}
	
}

File: testresources/csharp/benchmark/Domain/Direct/Violating/CallInstanceInnerInterface.cs
namespace Domain.Direct.Violating
{
	using Domain.Direct;
	using Technology.Direct.Dao;

	public class CallInstanceInnerInterface
	{
		CallInstanceOuterClassDAO.CallInstanceInnerInterfaceDAO innerInterfaceDao;

		public CallInstanceInnerInterface(){}
		
		public void Test()
		{
			innerInterfaceDao.InterfaceMethod();
		}
	}
}

